src/intervals/intervals.jl

```diff
@@ -5,37 +5,34 @@
 
 ## Interval type
 
+const validity_check = false
```

Comment:
I think the default value should be `true`, since that is the case that (eventually) will ensure compliance with the standard. This could be changed by the `IA_VALID` environment variable, which I guess can be set also in the .juliarc file.